command/assets/example-short.nomad

```diff
@@ -16,24 +16,6 @@ job "example" {
       resources {
         cpu    = 500
         memory = 256
-
-        network {
```

Comment:
I think you are going to still need the dynamic port.

Reply:
the network stanza is good to keep, as `db` port is used above; but let's remove `service` stanza here.